cmd/run.go

```diff
@@ -118,6 +117,6 @@ func startHotReloadProcess(cmdFlutterAttach *exec.Cmd, buildTargetMainDart strin
 	}
 	err := cmdFlutterAttach.Start()
 	if err != nil {
-		fmt.Printf("hover: flutter attach failed: %v Hotreload disabled\n", err)
+		log.Warn("Flutter attach failed: %v Hotreload disabled", err)
```

Comment:
`flutter attach` is a command, which is why it was spelled lowercase. Maybe change to "The command 'flutter attach' failed: ....etc" ?